addons/sale/models/res_config_settings.py

```diff
@@ -127,6 +130,11 @@ def _onchange_quotation_validity_days(self):
                 'warning': {'title': "Warning", 'message': "Quotation Validity is required and must be greater than 0."},
             }
 
+    @api.onchange('quotation_confirmation_percentage')
+    def _onchange_quotation_confirmation_percentage(self):
+        if self.quotation_confirmation_percentage <= 0 or self.quotation_confirmation_percentage > 100.0:
```

Comment:
with constraint on min/max quotation_confirmation_percentage, you won't need this onchange